tablet-report-parser/moses.pl

```diff
@@ -805,14 +805,22 @@ sub Create_Views{
     SELECT '$0' as program, '$VERSION' as version, '$opt{STARTTIME_PRINTABLE}' AS run_on, '$opt{LOCALHOST}' as host;
 
   CREATE VIEW tablets_per_node AS 
-    SELECT node_uuid,min(public_ip) as node_ip,min(region ) as region,  count(*) as tablet_count,
-           sum(CASE WHEN private_ip = leader THEN 1 ELSE 0 END) as leaders,
+    SELECT node_uuid,ip as node_ip,zone,  count(*) as tablet_count,
```

Comment:
Why the change from `public_ip` to `ip` here?

Similarly, why the change from `region` to `zone`? One region may have n zones and each zone belongs to one region.

Reply:
Thanks for catching this - this was a mistake - it is being reverted, and corrected.